source/module_base/memory.cpp

```diff
@@ -6,17 +6,17 @@
 
 namespace ModuleBase
 {
-// 1024 bit  = 1 Byte
+//    8 bit  = 1 Byte
 // 1024 Byte = 1 KB
 // 1024 KB   = 1 MB
 // 1024 MB   = 1 GB
 double Memory::total = 0.0;
-double Memory::complex_matrix_memory = 16.0;//(Byte)
-double Memory::double_memory = 8.0;//(Byte)
-double Memory::int_memory = 4.0;//(Byte)
-double Memory::bool_memory = 1.0;//(Byte)
-double Memory::float_memory = 4.0;//(Byte)
-double Memory::short_memory = 2.0;//(Byte)
+double Memory::complex_matrix_memory = 2*sizeof(double); //16.0 (Byte)
```

Comment:
Why these memory definitions are in `double`?

Reply:
double for easier addition, I guess